src/raygun.rum.js

```diff
@@ -155,12 +155,10 @@ var raygunRumFactory = function(window, $, Raygun) {
     };
 
     this.endSession = function() {
```

Comment:
This looks good to me but since this not only ends the current session, it also starts a new one I think we need to come up with a name that better describes what the method does so it is super clear to our users.

Reply:
I do agree with this, but from looking at other providers they also use `endSession`. While it's not ideal I think it's better to be consistent